common/source/docs/common-imu-fft.rst

```diff
@@ -0,0 +1,107 @@
+.. _common-imu-fft:
+
+[copywiki destination="copter,plane"]
+
+======================================
+Managing Gyro Noise with In-Flight FFT
+======================================
+
+*Note that this feature will be released in Copter/Plane 4.1*
+
+As :ref:`discussed<common-imu-notch-filtering>`, the effect of motor vibration on gyros in ArduPilot autopilot installations can be managed effectively through the use of targeted notch filtering. Dynamic notch filtering is most effective when the dynamic notch center frequency can be driven by ESC telemetry, but not all aircraft have access to ESC telemetry or the ESC telemetry is not entirely representative of the dominant noise.
+One alternative to ESC telemetry is to measure the noise peak directly through the use of digital signal processing techniques, most notably the Fast Fourier Transform or FFT.
+
```

Comment:
Add the paragraph below for tradheli
For helicopters, use of this feature depends on how well the rotor speed is being held.  Any helicopter using an ESC governor will not need this feature even if the ESC cannot provide arducopter rotor speed.  ESC governors hold rotor speed very well and the harmonic notch frequency can be set to the rotor speed set in the ESC governor.  If the internal rotor speed controller governor is used, which requires an RPM sensor, then it is recommended that the harmonic notch use the measured RPM to set the notch frequency.  This feature would be most useful for throttle curve method for controlling rotor speed.

Reply:
Thanks done